O/OptimPack/build_tarballs.jl

```diff
@@ -37,8 +37,8 @@ products = [
 ]
 
 # Dependencies that must be installed before this package can be built
-dependencies = [
+dependencies = Dependency[
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Comment:
Are you OK with requiring Julia v1.6?

Reply:
Yes if that's needed.  Thanks for the fix.